go/src/koding/kites/kloud/provider/azure/stack.go

```diff
@@ -99,6 +101,14 @@ func (s *Stack) VerifyCredential(c *stack.Credential) error {
 	return nil
 }
 
+func trim(s string, n int) string {
```

Comment:
Done, changed to `substringN` - `trimN` could confuse it trims N last characters, not all the characters above N.